sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/PartitionProcessorImpl.java

```diff
@@ -109,6 +115,7 @@ public PartitionProcessorImpl(ChangeFeedObserver observer,
                     .getToken();
 
                 if (documentFeedResponse.getResults() != null && documentFeedResponse.getResults().size() > 0) {
+                    logger.info("Partition {}: processing {} feeds with owner {}.", this.lease.getLeaseToken(), documentFeedResponse.getResults().size(), this.lease.getOwner());
```

Comment:
is logger level info intentional on every feed response ?

Reply:
yes, for every batch received; we had "complains" that CFP did not detect/process certain feeds in the past.